resume-buddy/src/main/java/com/google/sps/servlets/BlobStoreServlet.java

```diff
@@ -0,0 +1,26 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * When the fetch() function requests the /blobstore-upload-url URL, the content of the response is
+ * the URL that allows a user to upload a file to Blobstore.
+ */
+@WebServlet("/blobstore-upload-url")
```

Comment:
You don't have any JavaScript actually triggering this BlobstoreServlet to do anything.

Ani wrote a function called [`blobUpload()`](https://github.com/AniMachaidze/step/blob/46692a06618be85f4ec2b59cd06cc4f7f519b678/portfolio/src/main/webapp/script.js#L168) that fetches the upload URL from this servlet. Then it sets the upload URL to the action of the [comment form here](https://github.com/AniMachaidze/step/blob/46692a06618be85f4ec2b59cd06cc4f7f519b678/portfolio/src/main/webapp/career.html#L140).

You will need to write a similar JS function that fetches the upload URL for the PDF and sets it to the action of the reviewee submission form. 

@AniMachaidze do you have additional comments/clarification on how `blobUpload()` works?

Reply:
Okay, I will look at Ani's and write the new JS function!

Reply:
Follow up question @oydang : For setting the action, I'm I doing this to the form that is already existing in reviewee-form.html correct?

Reply:
Yes, specifically this form: https://github.com/googleinterns/ResumeBuddy/blob/9f26ed3c437e083e4b5bdff21b6d885c3e699447/resume-buddy/src/main/webapp/reviewee-form.html#L12